packages/components/organisms/f-footer/src/components/LinkList.vue

```diff
@@ -97,6 +101,17 @@ export default {
 
         testId () {
             return this.panelCollapsed ? 'linkList-wrapper-collapsed' : 'linkList-wrapper';
+        },
+
+        /**
+         * Removes euro2020 links from the footer if `showEuro2020Links` prop is false
+         * by eliminating links with type 'euro2020' from link list
+         */
+        filteredLinkList () {
+            if (this.showEuro2020Links) {
+                return this.linkList.links;
+            }
+            return this.linkList.links.filter(link => link.type !== 'euro2020');
```

Comment:
Matter of preference but it doesn't matter :D

```suggestion
           return (this.showEuro2020Links)
             ? this.linkList.links
             : this.linkList.links.filter(link => link.type !== 'euro2020');
```